flow.json

```diff
@@ -7,43 +14,64 @@
   },
   "deployments": {
     "emulator": {
-      "emulator-owner": ["FiatTokenInterface", "OnChainMultiSig"],
-      "emulator-allowance": [],
-      "emulator-blocklister": [],
-      "emulator-minter": [],
-      "emulator-minterController1": [],
-      "emulator-minterController2": [],
-      "emulator-non-allowance" : [],
-      "emulator-non-blocklister":[],
-      "emulator-non-minter": [],
-      "emulator-non-multisig-account": [],
-      "emulator-non-pauser": [],
-      "emulator-non-vaulted-account": [],
-      "emulator-pauser": [],
-      "emulator-vaulted-account": [],
-      "emulator-w-1000": [],
-      "emulator-w-500-1": [],
-      "emulator-w-500-2": [],
-      "emulator-w-250-1": [],
-      "emulator-w-250-2": []
+      "emulator-owner": ["FiatToken"]
     }
   },
   "contracts": {
     "FungibleToken": {
-      "source": "./contracts/FungibleToken.cdc",
+      "source": "./contracts/utility/FungibleToken.cdc",
       "aliases": {
         "emulator": "0xee82856bf20e2aa6",
+        "testing": "0x0000000000000002",
+        "previewnet": "0xa0225e7000ac82a9",
         "testnet": "0x9a0766d93b6608b7",
         "mainnet": "0xf233dcee88fe0abe"
       }
     },
-    "FiatTokenInterface": "./contracts/FiatTokenInterface.cdc",
-    "OnChainMultiSig": "./contracts/OnChainMultiSig.cdc"
-  },
-  "networks": {
-    "emulator": "127.0.0.1:3569",
-    "mainnet": "access.mainnet.nodes.onflow.org:9000",
-    "testnet": "access.devnet.nodes.onflow.org:9000"
+    "Burner": {
+      "source": "./contracts/utility/Burner.cdc",
+      "aliases": {
+        "emulator": "0xf8d6e0586b0a20c7",
+        "testing": "0x0000000000000001",
+        "previewnet": "0xa0225e7000ac82a9",
```

Comment:
BTW I think this is supposed to be `b6763b4399a888c8`

```
alex.zaman flow-usdc[feature/cadence-1*+] %flow-c1 accounts get b6763b4399a888c8 --network=previewnet
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
                           ⚠ Upgrade to Cadence 1.0                           
     The Crescendo network upgrade, including Cadence 1.0, is coming soon.     
     You may need to update your existing contracts to support this change.     
                     Please visit our migration guide here:                     
             https://cadence-lang.org/docs/cadence_migration_guide             
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

Address  0xb6763b4399a888c8
Balance  955996688.84853215
Keys     1

Key 0   Public Key               8666829b5bb860455945455bc9c0383af2647d6500aa92bc4d6119e828b4070d5c7f6ef4180e8bd4cf0c92c1e346c711c6fdbbb87e6b0f5e860222b22dfddfab
        Weight                   1000
        Signature Algorithm      ECDSA_secp256k1
        Hash Algorithm           SHA2_256
        Revoked                  false
        Sequence Number          369
        Index                    0

Contracts Deployed: 16
Contract: 'FlowClusterQC'
Contract: 'RandomBeaconHistory'
Contract: 'FlowStorageFees'
Contract: 'FlowDKG'
Contract: 'LockedTokens'
Contract: 'ViewResolver'
Contract: 'EVM'
Contract: 'NonFungibleToken'
Contract: 'FlowStakingCollection'
Contract: 'FlowServiceAccount'
Contract: 'Burner'
Contract: 'FlowEpoch'
Contract: 'StakingProxy'
Contract: 'MetadataViews'
Contract: 'FlowIDTableStaking'
Contract: 'NodeVersionBeacon'


Contracts (hidden, use --include contracts)

alex.zaman flow-usdc[feature/cadence-1*+] %flow-c1 accounts get a0225e7000ac82a9 --network=previewnet
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
                           ⚠ Upgrade to Cadence 1.0                           
     The Crescendo network upgrade, including Cadence 1.0, is coming soon.     
     You may need to update your existing contracts to support this change.     
                     Please visit our migration guide here:                     
             https://cadence-lang.org/docs/cadence_migration_guide             
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

Address  0xa0225e7000ac82a9
Balance  1000.00082026
Keys     1

Key 0   Public Key               8666829b5bb860455945455bc9c0383af2647d6500aa92bc4d6119e828b4070d5c7f6ef4180e8bd4cf0c92c1e346c711c6fdbbb87e6b0f5e860222b22dfddfab
        Weight                   1000
        Signature Algorithm      ECDSA_secp256k1
        Hash Algorithm           SHA2_256
        Revoked                  false
        Sequence Number          5
        Index                    0

Contracts Deployed: 3
Contract: 'FungibleTokenMetadataViews'
Contract: 'FungibleTokenSwitchboard'
Contract: 'FungibleToken'


Contracts (hidden, use --include contracts)
```

Reply:
Good catch!